doc/guide/authentication.xml

```diff
@@ -72,31 +96,54 @@
     <section id="public-key">
       <title>Public key</title>
 
-    <para>When you successfully log into the primary server, an <filename>ssh-agent
-      </filename> is started. The following keys are then preloaded into the
-      <filename>ssh-agent</filename> provided they are supported by your SSH
-      version, present, with the correct permissions, and either unencrypted
-      or encrypted with the same password that was used to login.</para>
-
-<programlisting>
-~/.ssh/id_rsa
-~/.ssh/id_dsa
-~/.ssh/id_ed25519
-~/.ssh/id_ecdsa
-</programlisting>
+    <para>When you successfully log into the primary server, a
+    <filename>ssh-agent</filename> is started and keys are loaded into
+    it by running <filename>ssh-add</filename> without any arguments.
+    Any passphrase prompt is answered with the password used to log
+    into the primary server.</para>
 
-    <para>Cockpit provides an interface for loading other keys into the agent
+    <para>Cockpit provides a user interface for loading other keys into the agent
       that could not be automatically loaded.</para>
 
-    <para>The target server will need to have public key authentication enabled in
-      <filename>sshd</filename>, and the public key you wish to use must be present in
-      <filename>~/.ssh/authorized_keys</filename>.</para>
+    <para>The target server will need to have public key
+    authentication enabled in <filename>sshd</filename>, and the
+    public key you wish to use must be present in
+    <filename>~/.ssh/authorized_keys</filename>.  Cockpit has a user
```

Comment:
Double space before a capital letter. Does it have a meaning in markdown?

Reply:
No, it's just habit.. I try to kick it, which means I am super inconsistent with myself now.

Reply:
Fixed.